Firestore/core/test/unit/bundle/bundle_serializer_test.cc

```diff
@@ -924,6 +926,38 @@ TEST_F(BundleSerializerTest, DecodesOrderBys) {
   VerifyNamedQueryRoundtrip(original);
 }
 
+// By default, the queries used for testing in this file always have default
+// OrderBy ("__name__") generated. We need to explicitly remove that for this
+// test.
+TEST_F(BundleSerializerTest, DecodeMissingOrderBysWorks) {
+  // This is `NamedQueryToJson(testutil::Query("bundles/docs/colls"))` with
+  // orderBy field manually removed.
+  auto json_string = R"|(
+{"name":"query-1",
```

Comment:
Nit: add a newline between the opening curly brace and `"name":`. Feel free to do in a follow-up if you don't want to retest this PR.

Reply:
Done.